src/main/java/ch01/racinggame/domain/Car.java

```diff
@@ -14,8 +15,10 @@ public Car(String name) {
         progressCnt = initCarCount;
     }
 
-    public void move() {
-        progressCnt++;
+    public void move(RandomNumber randomNumber) {
+        if(randomNumber.movable()){
```

Comment:
내부에서 스스로 판단할 수 있도록 하여 테스트 용이 👍 
```suggestion
    public void move(RandomNumber randomNumber) {
        if (randomNumber.movable()) {
```